package.json

```diff
@@ -48,7 +48,7 @@
     "less": "~1.7.3",
     "lynx": "~0.1.1",
     "lz-string": "~1.3.3",
-    "memcached": "git://github.com/3rd-Eden/node-memcached#10733b0e487793dde1d3a4f9201b1ec41c0be0c6",
+    "memcached": "~2.2.1",
```

Comment:
I'm wary of this change, since we had to point to a specific commit that the system relied on. The big problem is: there's no decent tests around memcache, etc. :(